Source/WebCore/en.lproj/Localizable.strings

```diff
@@ -778,6 +778,12 @@
 /* Validation message for input form controls requiring a constrained value according to pattern */
 "Match the requested format" = "Match the requested format";
 
+/* Validation message for input form controls requiring a constrained value according to pattern followed by a website-provided description of the pattern */
+"Match the requested format: %@" = "Match the requested format: %@";
+
+/* Validation message for input form controls requiring a constrained value according to pattern followed by a website-provided description of the pattern */
+"Match the requested format: %s" = "Match the requested format: %s";
```

Comment:
While this was copypasta, the fact that `update-webkit-localizable-strings` ends up doing the exact same makes me inclined to leave this as-is.

Reply:
@WebKit/glib-reviewers Is this string needed for localization on your platforms?

Reply:
Yes: it's displayed to users so it needs to be localized. Marking it with WEB_UI_FORMAT_STRING is all you need to do to achieve that.

This Localizable.strings is used only by Apple and Windows platforms, but you might as well leave it since it will just be added again next time the script is run.

Reply:
Thanks for confirming! We should probably fix the script at some point...

Reply:
Filed https://bugs.webkit.org/show_bug.cgi?id=259519.